tests/src/JIT/Regression/JitBlue/GitHub_5047/project.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I do think that this can probably get away with the minimal project.json, however.